website/docs/d/waf_rate_based_rule.html.markdown

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Same here -- can you please add a sidebar link to this page in the `website/aws.erb` file? Thanks!